apps/src/templates/projects/ProjectWidgetWithData.jsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, {PropTypes} from 'react';
 import ProjectWidget from '@cdo/apps/templates/projects/ProjectWidget';
 import $ from 'jquery';
 
 const ProjectWidgetWithData = React.createClass({
   propTypes: {
-    projectTypes: React.PropTypes.arrayOf(React.PropTypes.string),
-    projectList: React.PropTypes.array
+    projectTypes: PropTypes.arrayOf(React.PropTypes.string),
+    projectList: PropTypes.array,
+    isRtl: PropTypes.bool.isRequired
```

Comment:
This component also gets used in StageExtras. Can we either update it their (if CSF has a rtl mode) or make this prop not required (default to false)?